src/main/java/ru/ayaz/ChatServer.java

```diff
@@ -0,0 +1,139 @@
+package ru.ayaz;
+
+
+import javax.jws.soap.SOAPBinding;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.*;
+import java.util.*;
+
+/**
+ * Created by ayaz on 11/4/16.
```

Comment:
I think, it is better to get rid of this comment, IDEA inserts it by default after your create new class through its interface, but I think its kind of stupid, cause you should use git and not comments to track your history.